x-pack/plugins/ml/public/application/components/anomaly_detection_empty_state/anomaly_detection_empty_state.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiButton,
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiLink,
+  EuiTitle,
+} from '@elastic/eui';
+import adImage from './ml_anomaly_detection.png';
+import { ML_PAGES } from '../../../../common/constants/locator';
+import { useMlKibana, useMlLocator, useNavigateToPath } from '../../contexts/kibana';
+import { checkPermission } from '../../capabilities/check_capabilities';
+import { mlNodesAvailable } from '../../ml_nodes_check';
+
+export const AnomalyDetectionEmptyState: FC = () => {
```

Comment:
for consistency with he DFA empty state component, this should live inside the jobs list component.
https://github.com/elastic/kibana/tree/main/x-pack/plugins/ml/public/application/jobs/jobs_list/components

Reply:
Moved in https://github.com/elastic/kibana/pull/123462/commits/3887de8d81314602434afcfef47101f8f1117416